tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/effects/debugger_effects_test.ts

```diff
@@ -708,6 +709,76 @@ describe('Debugger effects', () => {
         }
       );
     }
+
+    for (const dataAlreadyExists of [false, true]) {
+      it(
+        `scrolling to execution index triggers execution digest loading: ` +
+          `dataAlreadyExists=${dataAlreadyExists}`,
+        () => {
+          const originalScrollBeginIndex = 50;
+          const newScrollBeginIndex = originalScrollBeginIndex + 2;
+          const numExecutions = 100;
+          const displayCount = 10;
+          const pageSize = 20;
+          store.overrideSelector(getActiveRunId, runId);
+          store.overrideSelector(
+            getExecutionScrollBeginIndex,
+            newScrollBeginIndex
+          );
+          store.overrideSelector(getNumExecutions, numExecutions);
+          store.overrideSelector(getDisplayCount, displayCount);
+          store.overrideSelector(getExecutionPageSize, pageSize);
+          const pageLoadedSizes: {[pageIndex: number]: number} = {
+            0: 20,
+            1: 20,
+            2: 20,
+          };
+          if (dataAlreadyExists) {
+            pageLoadedSizes[3] = 10;
+          }
```

Comment:
instead of logic inside test, just parameterize pageLoadSize too. Otherwise, I will expect to have test on tests.

Reply:
Done.